lms/static/scripts/frontend_apps/utils/content-item.js

```diff
@@ -48,3 +45,21 @@ export function contentItemForLmsFile(ltiLaunchUrl, file) {
     file_id: file.id,
   });
 }
+
+/**
+ * Return a JSON-LD `ContentItem` representation of the LTI activity launch URL
+ * for a VitalSource ebook.
+ *
+ * @param {string} ltiLaunchUrl
+ * @param {string} bookId - VitalSource book ID (aka. `vbid`)
+ * @param {string} cfi -
+ *   Location in the book. This is an EPUB CFI path without the surrounding
+ *   `epubcfi(...)` fragment. See http://idpf.org/epub/linking/cfi/epub-cfi.html.
+ */
+export function contentItemForVitalSourceBook(ltiLaunchUrl, bookId, cfi) {
+  return contentItemWithParams(ltiLaunchUrl, {
+    vitalsource_book: 'true',
+    book_id: bookId,
+    cfi,
+  });
```

Comment:
The URL params here mirror those used for Canvas files. However I think we should consider replacing the boolean `vitalsource_book`, `canvas_file` params to indicate the type with an enum in future (eg. `type=canvas_file`, `type=vitalsource_book`). Note that if we do make this change, we'd have to support the older params indefinitely in the backend because existing assignments in Canvas or other LMSes may reference them.